hydjet.spec

```diff
@@ -0,0 +1,20 @@
+### RPM external hydjet 1.9.1
+
+Source: http://cern.ch/lokhtin/hydro/%{n}-%{realversion}.tar.gz
+
+BuildRequires: cmake
+
+Requires: pyquen pythia6 lhapdf
+
+
+%prep
+%setup -q -n %{n}-%{realversion}
+
+%build
+
+cmake . -DCMAKE_INSTALL_PREFIX=%i -DCMAKE_BUILD_TYPE=Release -DPYQUEN_DIR=${PYQUEN_ROOT} -DPYTHIA6_DIR=${PYTHIA6_ROOT} -DLHAPDF_ROOT_DIR=${LHAPDF_ROOT}
+cmake --build . --clean-first -- %{makeprocesses}
+
+%install
+
+cmake --build . --target install --clean-first -- %{makeprocesses}
```

Comment:
please remove `--clean-first` here otherwise it will clean and re-build

Reply:
Yes, that was the goal (to clean in case the previous build found). What's wrong? The same behaviour was realized for pyquen.

Reply:
there is no need to do twice. It has been done in the %build section

Reply:
I see, thanks!